src/sentry/api/bases/organization.py

```diff
@@ -190,9 +193,23 @@ def get_projects(
         :return: A list of Project objects, or raises PermissionDenied.
         """
         if project_ids is None:
-            project_ids = self.get_requested_project_ids_unchecked(request)
+            slugs = project_slugs or set(list(filter(None, request.GET.getlist("projectSlug"))))
```

Comment:
can this just be `set(request.GET.getlist("projectSlug")`?

Reply:
Oops - i should remove the `list` yea lol

Reply:
The filter is necessary since it's still possible to pass an empty string or something in the slug in which case we wouldn't be falling back to the id's